hack/tests/e2e-helm.sh

```diff
@@ -10,20 +10,24 @@ ROOTDIR="$(pwd)"
 TMPDIR="$(mktemp -d)"
 trap_add 'rm -rf $TMPDIR' EXIT
 
+test_namespace="test-e2e-helm"
+
 deploy_operator() {
-    kubectl create -f "$OPERATORDIR/deploy/service_account.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/role.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/role_binding.yaml"
     kubectl create -f "$OPERATORDIR/deploy/crds/helm.example.com_nginxes_crd.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/operator.yaml"
+    kubectl create -f "$OPERATORDIR/deploy/service_account.yaml"
+    kubectl create -f "$OPERATORDIR/deploy/cluster_role.yaml"
```

Comment:
The addition of `--namespace` below makes sense, but why is the test operator now cluster-scoped? Shouldn't its manifests just have a different namespace?

Reply:
The change to the `operator.yaml` is to make it watch all namespaces so that it can see CRs outside of its own namespace. Not sure if that answers your question...